graph/network/distance_example_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright ©2025 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package network_test
+
+import (
+	"fmt"
+
+	"gonum.org/v1/gonum/graph/network"
+	"gonum.org/v1/gonum/graph/path"
+	"gonum.org/v1/gonum/graph/simple"
+)
+
+// ExampleEccentricity_diameter shows how to compute the diameter of a graph
+// based on its eccentricity map.
+func ExampleEccentricity_diameter() {
```

Comment:
```suggestion
func ExampleEccentricity_diameter() {
```
We don't need to have a godoc for the function.